noark-extraction-validator/src/main/resources/noark53/noark53-validation.xml

```diff
@@ -356,58 +370,556 @@
+        </queries>
+    </check>
+
+    <check id="ASC11">
+        <title>Document Descriptions without Document Objects</title>
+        <description>Number of Document Descriptions grouped by Series</description>
+        <group>arkivstruktur</group>
+        <queries>
+            <info>
+                <![CDATA[
+                    SELECT r._ref_arkivdel series_system_id, COUNT(*) document_descriptions_without_document_objects
+                    FROM arkivstruktur.dokumentbeskrivelse d
+                    JOIN arkivstruktur.registrering r ON r.systemid = d._ref_registrering
+                    WHERE d.systemid NOT IN (
+                        SELECT _ref_dokumentbeskrivelse
+                        FROM arkivstruktur.dokumentobjekt
+                        WHERE _ref_dokumentbeskrivelse IS NOT NULL
+                        GROUP BY _ref_dokumentbeskrivelse
+                    )
+                    GROUP BY r._ref_arkivdel;
+                ]]>
+            </info>
+        </queries>
+    </check>
+
+    <check id="ASC12">
+        <title>Document Object count</title>
+        <description>Number of Document Objects grouped by Series and parent type (Document Description or Record)</description>
+        <group>arkivstruktur</group>
+        <queries>
+            <info>
+                <![CDATA[
+                    SELECT
+                        r._ref_arkivdel series_system_id,
+                        COUNT(*) document_objects,
+                        (d._ref_dokumentbeskrivelse IS NOT NULL) in_document_description,
+                        (d._ref_dokumentbeskrivelse IS NULL) in_record
+                    FROM arkivstruktur.dokumentobjekt d
+                    LEFT JOIN arkivstruktur.dokumentbeskrivelse dd ON dd._id = d._parent_id
+                    LEFT JOIN arkivstruktur.registrering r ON r._id = d._parent_id
+                    GROUP BY series_system_id, in_document_description, in_record
+
+                    UNION ALL
+
+                    SELECT 'total', count(*), TRUE in_document_description, TRUE in_record
+                    FROM arkivstruktur.dokumentobjekt;
+                ]]>
+            </info>
+        </queries>
+    </check>
+
+    <check id="ASC13">
+        <title>Case Party count</title>
+        <description>Number of Case Parties grouped by Series</description>
+        <group>arkivstruktur</group>
+        <queries>
+            <info>
+                <![CDATA[
+                    SELECT f._ref_arkivdel series_system_id, COUNT(*) number_of_case_parties
+                    FROM arkivstruktur.sakspart cp
+                    JOIN arkivstruktur.mappe f ON f.systemid = cp._ref_mappe
+                    GROUP BY f._ref_arkivdel;
+                ]]>
+            </info>
+        </queries>
+    </check>
+
+    <check id="ASC14">
+        <title>Note count</title>
+        <description>Number of Notes related to Files, Records, and Document Descriptions grouped by Series</description>
```

Comment:
`Number of notes associated with files, records, or document descriptions, grouped by series`